### PR TITLE
GH-81: [feat] delete event

### PR DESCRIPTION
**Pull request by wolfie7679**
- Created the delete event endpoint
- Created unit tests associated with endpoint

### CODE REVIEW

File: Microservices/event-service/src/main/java/app/sportahub/eventservice/service/event/EventServiceImpl.java [hunk 1]
```diff
@@ -45,4 +45,19 @@ public EventResponse createEvent(EventRequest eventRequest) {
...
+     * @throws EventDoesNotExistException if there is no event associated with the provided id
+     */
+    @Override
+    public void deleteEvent(String id) {
+
+        Event evt = eventRepository.findEventById(id).orElseThrow(() -> new EventDoesNotExistException(id));
+        eventRepository.delete(evt);
+        log.info("deleteEvent: Event with id: {} was successfully deleted", id);
+
+
```
Comment (walidoow) [inline Microservices/event-service/src/main/java/app/sportahub/eventservice/service/event/EventServiceImpl.java:61]: Can you delete these trailing spaces?

**Review by walidoow — CHANGES_REQUESTED** [review]
Aside from the trailing spaces, everything looks good, I will accept the PR once you make that change. Good work!

File: Microservices/event-service/src/main/java/app/sportahub/eventservice/service/event/EventServiceImpl.java (hunk 1 above)
Comment (wolfie7679) [reply Microservices/event-service/src/main/java/app/sportahub/eventservice/service/event/EventServiceImpl.java:61]: sure

**Review by walidoow — APPROVED** [review]
Looks good to me! Great first PR 🥳

**Review by Ziadsharkos — APPROVED** [review]
Looks good to me!

**Pull request merged**